modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/emodelgeneration/v0.1.0.json

```diff
@@ -1,46 +0,0 @@
-{
```

Comment:
I would recommand not to remove schemas for the repository but to keep them and add the annotation:

"@type": "nxv:Schema",
"owl:deprecated": true

Reply:
I had to remove it to know how this should be handled :) thanks for the suggestion.